multilayer_EC_glioma_2023/Readme.md

```diff
@@ -0,0 +1,48 @@
+## The longitudinal relation between executive functioning and multilayer network topology in glioma patients
+### Marike R van Lingen, Lucas C Breedt, Jeroen J G Geurts, Arjan Hillebrand, Martin Klein, Mathilde C M Kouwenhoven, Shanna D Kulik, Jaap C Reijneveld, Cornelis J Stam, Philip C De Witt Hamer, Mona L M Zimmermann, Fernando A N Santos, Linda Douw
+
+Scripts used in doi: 10.1007/s11682-023-00770-w.
```

Comment:
maybe make this a hyperlink?

Reply:
@mrvanlingen added some little changes to be made!

Reply:
Check :) Good suggestion!